app/javascript/packs/puerto-rico-hint.ts

```diff
@@ -0,0 +1,14 @@
+import { t } from '@18f/identity-i18n';
+
+const selector = document.getElementById('in_person_address_state');
+
+selector?.addEventListener('change', (event) => {
+  const { value } = event?.target as HTMLFormElement;
+  if (value === 'PR') {
+    const hints = document.querySelectorAll('.usa-hint');
+    const firstHint = hints[0];
+    const secondHint = hints[1];
```

Comment:
We can use destructuring for this
```suggestion
    const [firstHint, secondHint] = document.querySelectorAll('.usa-hint');
```